.circleci/config.yml

```diff
@@ -16,8 +16,6 @@ jobs:
       - run: cd server; go get -v -t -d ./...
       - run: cd server; go test -v ./...
 
-        # Make sure the formatting is right.
-      - run: cd js; sysconfcpus -n 2 elm-format --validate *.elm
```

Comment:
We can delete this altogether, if you want. I don't necessarily care about format being enforced.

Reply:
Sure